code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -186,138 +187,166 @@
-    new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
-    new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
+	..()
+	new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
+	new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
+	new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
+	new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
+	new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
+	new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
+	new /obj/item/weapon/reagent_containers/pill/spaceacillin(src)
 
 /obj/item/weapon/storage/pill_bottle/tramadol
 	name = "bottle of Tramadol pills"
 	desc = "Contains pills used to relieve pain."
 
-	New()
-		..()
-		new /obj/item/weapon/reagent_containers/pill/tramadol( src )
-		new /obj/item/weapon/reagent_containers/pill/tramadol( src )
-		new /obj/item/weapon/reagent_containers/pill/tramadol( src )
-		new /obj/item/weapon/reagent_containers/pill/tramadol( src )
-		new /obj/item/weapon/reagent_containers/pill/tramadol( src )
-		new /obj/item/weapon/reagent_containers/pill/tramadol( src )
-		new /obj/item/weapon/reagent_containers/pill/tramadol( src )
+/obj/item/weapon/storage/pill_bottle/tramadol/New()
+	..()
+	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
+	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
+	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
+	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
+	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
+	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
+	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
 
 /obj/item/weapon/storage/pill_bottle/citalopram
 	name = "bottle of Citalopram pills"
 	desc = "Contains pills used to stabilize a patient's mood."
 
-	New()
-		..()
-		new /obj/item/weapon/reagent_containers/pill/citalopram( src )
-		new /obj/item/weapon/reagent_containers/pill/citalopram( src )
-		new /obj/item/weapon/reagent_containers/pill/citalopram( src )
-		new /obj/item/weapon/reagent_containers/pill/citalopram( src )
-		new /obj/item/weapon/reagent_containers/pill/citalopram( src )
-		new /obj/item/weapon/reagent_containers/pill/citalopram( src )
-		new /obj/item/weapon/reagent_containers/pill/citalopram( src )
+/obj/item/weapon/storage/pill_bottle/citalopram/New()
+	..()
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+
+/obj/item/weapon/storage/pill_bottle/carbon
+	name = "bottle of Carbon pills"
+	desc = "Contains pills used to neutralise chemicals in the stomach."

```

Comment:
Very minor and not required to be changed, `neutralize` seems to be the usual standard.

Reply:
americentrism out